### ISSUE TITLE
The md->tex converter breaks the Phys. Rev. (and possibly other)  template

### ISSUE THREAD

**Issue by jochym** [executablebooks/mystmd#737]
The missing newline at the end of the comment delimiting the "acronyms & glossary" section breaks the Phys Rev template "acknowledgments" section.

## Description

The \begin{acknowledgments} line from the template gets "glued" at the end of the comment line and effectively disabled. This can be worked-around in the template, but probably affects also other templates and sections. The resulting .tex file looks as follows:
```
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%%%%%%%%%%%%%  acronyms & glossary  %%%%%%%%%%%%%
\printglossaries
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%\begin{acknowledgments}
```
## Proposed solution

Trivial: add a newline at the end of the delimiting comment. Unfortunately, I do not know in which part it is generated, so I cannot propose actual patch.

## Additional notes

This may affect multiple users/templates and generates a very frustrating "silent" bug in the otherwise correct myst document. This may be very discouraging for novice users. Particularly due to the fact that acknowledgments is a standard part of a PR paper and makes mystmd unusable for such a user/purpose.

**Comment by rowanc1** [comment]
Thank you for flagging this @jochym, we will aim to get a fix out fast for this.

**Comment by fwkoch** [comment]
This was addressed in #740 and is now released with mystmd v1.1.27!

Let us know if you still have any trouble with this.
